libbeat/monitoring/report/elasticsearch/elasticsearch.go

```diff
@@ -172,6 +172,7 @@ func makeReporter(beat beat.Info, settings report.Settings, cfg *common.Config)
 
 	pipeline, err := pipeline.New(
 		beat,
+		pipeline.Monitors{},
```

Comment:
Why pass an empty instance here? This removes the `x-pack.monitoring.output` metrics from each beat.

It's also a good idea to create a logger named `x-pack.monitoring`.